pages/common/qmmp.md

```diff
@@ -23,10 +23,10 @@
 
 `qmmp --previous`
 
-- Print the current volume:
+- Display the current volume:
 
 `qmmp --volume-status`
 
-- Increase or decrease the volume of the currently playing audio by 5 steps:
+- [inc]rease or [dec]rease the volume of the currently playing audio by 5%:
```

Comment:
Have you checked that 5 steps is equivalent to 5%?

Reply:
I tested this locally and also checked the manpage can confirm that 5 steps mean 5% here.

![image](https://github.com/tldr-pages/tldr/assets/26346867/e5bfa82e-8e1f-43a5-b698-0999aaf17a47)